README.md

```diff
@@ -262,6 +274,29 @@ FIRETWEET_MAIN_DIR=/path/to/firetweet/src/main make android-lib
 You can also override this environment variable if you want to use the
 [Flashlight Android Tester](https://github.com/getlantern/lantern-mobile-single-app-example) app.
 
+#### Creating the Android library without docker
+
+1. Install Java JDK 7 or 8
+2. Install [Android SDK Tools](http://developer.android.com/sdk/index.html#Other)
+3. Install NDK(http://developer.android.com/ndk/downloads/index.html)
+4. Install lantern's [fork of gomobile](https://github.com/getlantern/mobile)
```

Comment:
Maybe you encountered the same problem as https://github.com/getlantern/lantern/issues/3392? 

I'm on El Capitan (10.11.2) and Xcode 7.2(7C68).

```
clang -v
Apple LLVM version 7.0.2 (clang-700.1.81)
```